backend/question/src/services/get/types.ts

```diff
@@ -0,0 +1,75 @@
+import type { IServiceResponse } from '@/types';
+
+//=============================================================================
+// /get
+//=============================================================================
+export type IGetQuestionsPayload = {
+  // Filters
+  questionName?: string;
+  difficulty?: string;
+  topic?: string;
+  // Pagination
+  pageNum?: number; // Default 0
+  recordsPerPage?: number; // Default 20
+};
+
+export type IGetQuestionsResponse = IServiceResponse<{
+  // eslint-disable-next-line @typescript-eslint/no-empty-object-type
+  questions: Array<{
+    // TODO: Add schema from DB
+    /**
+     * - name
+     * - number
+     * - difficulty
+     * - topic
```

Comment:
To add to above comment, maybe the schema people can consider if they want to store it in one table/across multiple tables